core/src/main/java/com/linecorp/armeria/server/ParameterizedPathMapping.java

```diff
@@ -163,6 +182,26 @@ private static String paramName(String token) {
         return null;
     }
 
+    /**
+     * Return true if path parameter contains capture the rest path pattern {*foo}.
+     */
+    private static boolean isCaptureTheRestPathParam(String token) {
```

Comment:
We usually don't use `the` for method names.
How about?:
```suggestion
    private static boolean isMultipleSegmentMatching(String token) {
```

Reply:
I see.
As I commented here, multiple segments seem to indicate `**` e.g. `/service/**`.  It's different with this pattern `{*foo}`.
https://github.com/line/armeria/pull/3997#discussion_r774415719

So can we use `isCaptureRestPathMatching` or another name which indicates  `{*foo}` ?

Reply:
`isCaptureRestPathMatching` sounds good.